__tests__/app.test.js

```diff
@@ -208,4 +206,93 @@ describe('Get /api/reviews/:review_id/comments', () => {
             });
     });
     
-});
\ No newline at end of file
+});
+
+describe('POST /api/reviews/:review_id/comments', () => {
+    test("201: response with posted object", () => {
+        const newComment = {
+            username:"mallionaire",
+            body:"test"
+        }
+
+        return request(app)
+            .post("/api/reviews/1/comments")
+            .send(newComment)
+            .expect(201)
+            .then( res => {
+                const { comment } = res.body;
+                expect(comment).toBeInstanceOf(Object);
+                expect(comment).toMatchObject({
```

Comment:
There is no need to have a .toBeInstanceOf(Object) here as the .toMatchObject is doing this for you as well.